packages/analytics/lib/index.d.ts

```diff
@@ -651,26 +662,6 @@ export namespace FirebaseAnalyticsTypes {
      * @param enabled A boolean value representing whether Analytics collection is enabled or disabled. Analytics collection is enabled by default.
      */
     setAnalyticsCollectionEnabled(enabled: boolean): Promise<void>;
-
```

Comment:
Need to keep this in otherwise it's a hard deprecation for TS users. Add a `@deprecated` line on the code comments